pkg/kubelet/server/server.go

```diff
@@ -417,6 +419,20 @@ func (s *Server) InstallDebuggingHandlers(criHandler http.Handler) {
 	}
 }
 
+// InstallDebuggingDisabledHandlers registers the HTTP request patterns that provide better error message
+func (s *Server) InstallDebuggingDisabledHandlers() {
+	glog.Infof("Adding debug disabled handlers to kubelet server.")
+
+	h := http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		http.Error(w, "Server endpoints for log collection and local running of containers and commands are disabled.", http.StatusMethodNotAllowed)
+	})
+
+	paths := []string{"/run/", "/exec/", "/attach/", "/portForward/", "/containerLogs/", "/runningpods/"}
```

Comment:
This might as well include all the debug paths. You're missing: pprofBasePath, logsPath

Reply:
Done.